service/history/timerBuilder.go

```diff
@@ -42,6 +42,12 @@ const (
 	emptyTimerID = -1
```

Comment:
There were couple of places that i didn't change as it doesn't touch activity timers but anyway just removed it completely.